src/pages/track-record.js

```diff
@@ -4,34 +4,129 @@ import {graphql} from 'gatsby';
 
 import * as trackRecordStyles from './track-record.module.scss';
+import * as sectionStyles from './track-record-section.module.scss';
 import ClientImages from '../components/client-images/client-images.js';
 
-const TrackRecordPage = ({data, location}) => {
-  /*
-   * To change the track record sections text change
-   * content/data/track-record-sections.yaml
-   * allTrackRecordSectionsYaml queries the file
-   * Query is at bottom of the page
-   */
-  const sections = data.allTrackRecordSectionsYaml.edges;
+const TrackRecordPage = ({location}) => {
   return (
     <Layout location={location}>
       <SEO title="Track Record"/>
       <h1 className={trackRecordStyles.title}>Track record</h1>
       <div className={trackRecordStyles.content}>
-        {sections.map(({key, node}) => {
-          return (
-            <TrackRecordSection
-              key={node.id}
-              trackRecord={node}
+        <div className={sectionStyles.container}>
+          <h2 className={sectionStyles.title}>
+            <img
+              className={sectionStyles.icon}
+              alt={`Generative AI Solutions icon`}
+              src={`/track-record/icon1.svg`}
             />
-          );
-        })}
+            Generative AI Solutions
+          </h2>
+          <p>
+            Using LLMs and state-of-the-art techniques
+            such as multi-agent systems and advanced prompt
+            engineering to create generative AI solutions.
+          </p>
+          <b>Previous projects:</b>
+          <ul className={sectionStyles.projectList}>
+            <li>
+              <b><a
+                href="https://www.aisentr.com/"
+                target="_blank"
+                rel="noreferrer"
+              >AiSentr:</a></b> delivered prototypes using Azure GPTs, RAG and a
+              multi-agent framework, to help close an important partnership for
+              the business.
+            </li>
+            <li>
+              <b>Automatic Classifiers for Peace:</b> delivered an
+              LLM-powered classifier to aid peacebuilders in finding insights
+              from social media data. Open source repository and community <a
+                href="https://gitlab.com/automatic-classifiers-for-peace/acfp"
+                target="_blank"
+                rel="noreferrer"
+              >here</a>.
+            </li>
+          </ul>
+        </div>
+        <div className={sectionStyles.container}>
+          <h2 className={sectionStyles.title}>
+            <img
+              className={sectionStyles.icon}
+              alt={`End-to-End AI Platforms icon`}
+              src={`/track-record/icon2.svg`}
+            />
+            End-to-End AI Platforms
+          </h2>
+          <p>
+            Building and delivering complete AI platforms that covers the
+            full spectrum from data collection and processing, to data
+            visualization, derivation of insights, and automated
```

Comment:
I think this is missing "developing models" or "start-of-the-art ML model development" or something like that.

Reply:
a5e44c0